README.md

```diff
@@ -5,8 +5,7 @@ This repo is used to document research performed by the packit team.
 
 ## Topics covered
 
-* [First topic](/topic)
-*
+* [Automation tools](./automation-tools)
```

Comment:
please be more descriptive about the research, so that outsiders would know what is your topic about; "Automation tools" is really generic